src/controllers/customer.js

```diff
@@ -43,11 +43,24 @@ const createCustomer = async (req, res) => {
                 return res.status(409).json({ error: "A customer with the provided email already exists" })
             }
         }
-        
+
         res.status(500).json({ error: e.message })
     }
 }
 
-module.exports = {
-    createCustomer
-}
+const updateCustomer = async (req, res) => {
+    const customerId = Number(req.params.id);
+    const { name, phone, email } = req.body;
+  
+    const updatedCustomer = await prisma.customer.update({
+      where: {
+        id: customerId,
+      },
+      data: { name: name, phone: phone, email: email },
```

Comment:
Since the variables have the same name as the object properties, you can short-hand this:
```js
data: { name, phone, email }
```